packages/app/src/static-props/variants/get-international-variant-chart-data.ts

```diff
@@ -0,0 +1,83 @@
+import { In, InVariants } from '@corona-dashboard/common';
+import { isDefined } from 'ts-is-present';
+import { VariantChartValue } from './get-variant-chart-data';
+
+export function getInternationalVariantChartData(inData: Record<string, In>) {
+  const variantChartData = Object.fromEntries(
+    Object.entries(inData).map(([key, value]) => [
+      key,
+      getVariantChartData(value.variants),
+    ])
+  );
+
+  return { variantChartData } as const;
+}
+
+const EMPTY_VALUES = {
+  variantChart: null,
+  dates: {
+    date_of_insertion_unix: 0,
+    date_start_unix: 0,
+    date_end_unix: 0,
+  },
+} as const;
+
+export function getVariantChartData(inVariants: InVariants | undefined) {
+  if (!isDefined(inVariants) || !isDefined(inVariants.values)) {
```

Comment:
```suggestion
  if (!isDefined(inVariants?.values)) {
```
I think this is effectively the same

Reply:
Yes, I tried that, but unfortunately, the compiler then gives a 'can be undefined' error on the `inVariants` variable.